ch2.txt

```diff
@@ -30,6 +23,11 @@ CHAPTER 2. WAGING WAR
 
 16. Now in order to kill the enemy, our men must be roused to anger; that there may be advantage from defeating the enemy, they must have their rewards. 
 
+asdf
+asdf
+asdf
+adsf
+
 17. Therefore in chariot fighting, when ten or more chariots have been taken, those should be rewarded who took the first. Our own flags should be substituted for those of the enemy, and the chariots mingled and used in conjunction with ours. The captured soldiers should be kindly treated and kept. 
```

Comment:
How would it handle this? This looks like a scoped block, but could be intended
to be a single line comment. In particular because it starts from the previous
comment? e.g. I could be commenting from asdf to the line starting with 17. Or I could just comment about something on line 17.